src/GitHubActionsDisplay.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "@toast-ui/chart/dist/toastui-chart.css"
+import Chart from "@toast-ui/chart";
+
+export default class GitHubActionsDisplay extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  componentDidMount() {
+    this.update();
+  }
+  async update() {
+    const data = {
+      actions_statuses: JSON.parse(
+        await fetch(
+          "https://ossci-checks-status.s3.amazonaws.com/status.json",
```

Comment:
just for my own understanding: is this S3 file very frequently updated?

Reply:
the lambda runs every few minutes: https://console.aws.amazon.com/lambda/home?region=us-east-1#/functions/github-checks-status-updater?tab=code